tests/test-shortcodes-glossary.php

```diff
@@ -177,6 +177,23 @@ public function test_tooltipContent() {
 		$this->assertContains( '<div class="glossary">', $content );
 		$this->assertContains( $definitions[0], $content );
 		$this->assertContains( $definitions[1], $content );
+
+		// Initialize shortcode hooks
+		$this->_book();
+		\Pressbooks\Shortcodes\Complex\Complex::init();
+
+		// Testing glossary to support shortcodes
+		$content_with_shortcode = 'Evolutionary algorithms are a heuristic-based approach to solving problems that cannot be easily solved in polynomial time. [media src="https://www.youtube.com/watch?v=L--IxUH4fac" caption="Introduction to evolutionary algorithms" /]';
+		$content_with_shortcode = do_shortcode($content_with_shortcode);
```

Comment:
Something minor, just to stick to the WP convention, parameters inside functions must include one space between each parenthesis, e.g.: `do_shortcode( $content_with_shortcode );`

Reply:
The linter doesn't check test code.
Still. Carlos is right. 
CTRL-ALT-L in PhpStorm fixes it for me.

+ https://www.jetbrains.com/help/phpstorm/wordpress-aware-coding-assistance.html#configuring-wordpress-code-style
+ https://www.jetbrains.com/help/phpstorm/reformat-file-dialog.html

Reply:
I'll fix it and set the convention up.